### DISCUSSION TITLE
alarm

### DISCUSSION THREAD

**Discussion by MohammadSadeghMehrafzoon** [PPartisan/Simple-Alarms#24] [General]
Hi, I used your Alarm project, but it gives me this error

java.lang.nullpointerexception: attempt to invoke virtual method 'void androidx.appcompat.app.actionbar.setdisplayhomeasupenabled(boolean)' on a null object reference

**Comment by PPartisan** [comment]
Hi, have you made any modifications to the repo, like migrated to androidx or enabled jettifier? This import statement is for `androidx`, but this app still uses the support libraries.
